app/apps/client/extrernalComponent.js

```diff
@@ -0,0 +1,63 @@
+import { Meteor } from 'meteor/meteor';
+import { Session } from 'meteor/session';
+
+import { APIClient } from '../../utils';
+import { getUserAvatarURL } from '../../utils/lib/getUserAvatarURL';
+
+
+Meteor.startup(function() {
+	window.addEventListener('message', async ({ data, source }) => {
+		if (!data.hasOwnProperty('rcEmbeddedSDK')) {
+			return;
+		}
+
+		try {
+			const { action, id } = data.rcEmbeddedSDK;
+			const baseUrl = document.baseURI.slice(0, -1);
+
+			switch (action) {
+				case 'getUserInfo':
+					const { username, _id } = Meteor.user();
+					const avatarUrl = `${ baseUrl }${ getUserAvatarURL(username) }`;
+					source.postMessage({
+						rcEmbeddedSDK: {
+							action,
+							id,
+							payload: {
+								userId: _id,
+								username,
+								avatarUrl,
+							},
+						},
+					}, '*');
+					break;
+				case 'getRoomInfo':
+					const { name: roomName, _id: roomId } = Session.get(`roomData${ Session.get('openedRoom') }`);
+					let { members } = await APIClient.get('v1/groups.members', { roomId });
+
+					members = members.map(({ _id, username, status }) => ({
+						userId: _id,
+						username,
+						avatarUrl: `${ baseUrl }${ getUserAvatarURL(username) }`,
+						status,
```

Comment:
@lolimay do you recall why is the status property sent over to the game?

Reply:
I previously thought that the FRVR team should need this property to know the users' status. But there seems that they don't use it now. Should we remove this property?